PANCHANGE_data_preprocessing2020-07-15.Rmd

```diff
@@ -643,42 +660,78 @@ glad.mhq.raw.id <- glad.mhq.raw.id %>%
                     )
   )
 
-glad.mhq.raw.id %>%
+glad.mhd.raw.id %>%
   freq(agoraphobia_panic_disorder)
 ```
 
 ```{r Check agoraphobia_panic_disorder}
-glad.mhq.raw.id %>%
+glad.mhd.raw.id %>%
   select(
     agoraphobia_panic_disorder,
     agoraphobia_panic_disorder_numeric,
-    mhq.agoraphobia_numeric,
-    mhq.panic_disorder_numeric,
-    mhq.panic_attacks_numeric,
-    mhq.agoraphobia,
-    mhq.panic_disorder,
-    mhq.panic_attacks
+    mhd.agoraphobia_numeric,
+    mhd.panic_disorder_numeric,
+    mhd.panic_attacks_numeric,
+    mhd.agoraphobia,
+    mhd.panic_disorder,
+    mhd.panic_attacks
+  )
+```
+
+Depression and anxiety comorbidity
+```{r glad mhd depression_anxiety}
+glad.mhd.raw.id <- glad.mhd.raw.id %>%
+  mutate(
+    depression_and_anxiety_numeric =
+      case_when(
+        depressive_disorders_numeric == 1 &
+          anxiety_disorders_numeric == 1 ~ 0,
+        depressive_disorders_numeric == 1 &
+          anxiety_disorders_numeric == 1 ~ 1,
```

Comment:
I think there is a mistake here - they are both saying that depressive & anxiety disorders == 1

Reply:
changed